contracts/access/AccessControlDefaultAdminRules.sol

```diff
@@ -207,9 +249,27 @@ abstract contract AccessControlDefaultAdminRules is IAccessControlDefaultAdminRu
      * Internal function without access restriction.
      */
     function _beginDefaultAdminTransfer(address newAdmin) internal virtual {
-        _defaultAdminTransferDelayedUntil = SafeCast.toUint48(block.timestamp) + defaultAdminDelay();
+        (, uint48 delaySchedule) = pendingDefaultAdminDelay();
+        if (_isSet(delaySchedule)) _rollbackDefaultAdminDelay();
```

Comment:
Following my other suggestion, here we would just write:
```suggestion
        _materializeDefaultAdminDelay();
```

Reply:
No longer applies